multisig/src/lib.rs

```diff
@@ -42,6 +43,7 @@ pub trait Multisig:
         &self,
         esdt_safe_sc_address: ManagedAddress,
         multi_transfer_sc_address: ManagedAddress,
+        proxy_sc_address: ManagedAddress,
```

Comment:
because of this, the SC proxy contract will need to be deployed before the multisig. Corrected the integration tests. :heavy_check_mark: